$NetBSD: patch-ac,v 1.3 2010/02/10 03:55:18 dholland Exp $

Hack this brutally to accept the security-fixed apache 1.3 from 2/2010.
The differences from 1.3.41 to 1.3.42 are minimal.

--- pkg.sslmod/libssl.version~  2008-02-08 06:18:01.000000000 -0500
+++ pkg.sslmod/libssl.version   2010-02-09 05:29:19.000000000 -0500
@@ -1 +1 @@
-mod_ssl/2.8.31-1.3.41
+mod_ssl/2.8.31-1.3.42
